helm-chart/renku-gateway/templates/configmap.yaml

```diff
@@ -263,35 +228,35 @@ data:
           method = "drr"
           passHostHeader = false
           [[http.services.gitlab.LoadBalancer.servers]]
-            url = {{ .Values.gitlabUrl | default (printf "%s://%s/gitlab" (include "gateway.protocol" .) .Values.global.renku.domain) | quote }}
+            url = "{{ .Values.gitlabUrl | default (printf "%s://%s/gitlab" (include "gateway.protocol" .) .Values.global.renku.domain) }}""
             weight = 1
 
         [http.services.notebooks.LoadBalancer]
           method = "drr"
           passHostHeader = false
           [[http.services.notebooks.LoadBalancer.servers]]
-            url = {{ .Values.notebooks.hostname | default (printf "http://%s-notebooks" .Release.Name )  | quote }}
```

Comment:
All those local variables like `.Values.notebooks.hostname` aren't documented or used anywhere, let's just go with the default service names at the moment.